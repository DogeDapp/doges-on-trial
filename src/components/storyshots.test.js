import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'

initStoryshots({
  test: snapshotWithOptions({
    createNodeMock: element =>
      element.type === 'input' && element.props.type === 'file'
        ? document.createElement('input')
        : null
  })
})
