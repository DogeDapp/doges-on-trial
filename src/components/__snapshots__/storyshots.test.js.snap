// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button primary 1`] = `
<div
  data-radium={true}
  style={
    Object {
      "background": "rgb(255, 255, 255)",
      "bottom": 0,
      "display": "flex",
      "flexDirection": "column",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    data-radium={true}
    style={
      Object {
        "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 15,
        "paddingBottom": 15,
        "paddingTop": 2,
      }
    }
  >
    <h2
      data-radium={true}
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.6)",
          "fontSize": 20,
          "fontWeight": 200,
          "margin": 0,
          "padding": 0,
        }
      }
    >
      Kleros UI-Kit
    </h2>
  </div>
  <div
    data-radium={true}
    style={
      Object {
        "boxSizing": "border-box",
        "flex": 1,
        "margin": 50,
        "position": "relative",
      }
    }
  >
    <div
      data-radium={true}
      style={
        Object {
          "alignItems": "center",
          "bottom": 0,
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "center",
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <div
        data-radium={true}
        style={
          Object {
            "border": "solid 1px rgba(0, 0, 0, 0.0)",
            "boxSizing": "border-box",
            "height": "auto",
            "position": "relative",
            "width": "auto",
          }
        }
      >
        <div
          data-radium={true}
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": "html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block}audio:not([controls]){display:none;height:0}progress{vertical-align:baseline}[hidden],template{display:none}a{background-color:transparent;-webkit-text-decoration-skip:objects}a:active,a:hover{outline-width:0}abbr[title]{border-bottom:none;text-decoration:underline;text-decoration:underline dotted}b,strong{font-weight:inherit;font-weight:bolder}dfn{font-style:italic}h1{font-size:2em;margin:.67em 0}mark{background-color:#ff0;color:#000}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}img{border-style:none}svg:not(:root){overflow:hidden}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}figure{margin:1em 40px}hr{box-sizing:content-box;height:0;overflow:visible}button,input,optgroup,select,textarea{font:inherit;margin:0}optgroup{font-weight:700}button,input{overflow:visible}button,select{text-transform:none}[type=reset],[type=submit],button,html [type=button]{-webkit-appearance:button}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-cancel-button,[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-input-placeholder{color:inherit;opacity:.54}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}html{font:100%/1.45em 'Roboto';box-sizing:border-box;overflow-y:scroll;}*{box-sizing:inherit;}*:before{box-sizing:inherit;}*:after{box-sizing:inherit;}body{color:hsla(0,0%,0%,0.8);font-family:'Roboto';font-weight:normal;word-wrap:break-word;font-kerning:normal;-moz-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";-ms-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";-webkit-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";}img{max-width:100%;margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}h1{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:2rem;line-height:1.1;}h2{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1.51572rem;line-height:1.1;}h3{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1.31951rem;line-height:1.1;}h4{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1rem;line-height:1.1;}h5{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:0.87055rem;line-height:1.1;}h6{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:0.81225rem;line-height:1.1;}hgroup{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}ul{margin-left:1.45rem;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;list-style-position:outside;list-style-image:none;}ol{margin-left:1.45rem;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;list-style-position:outside;list-style-image:none;}dl{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}dd{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}p{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}figure{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}pre{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;font-size:0.85rem;line-height:1.45rem;}table{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;font-size:1rem;line-height:1.45rem;border-collapse:collapse;width:100%;}fieldset{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}blockquote{margin-left:1.45rem;margin-right:1.45rem;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}form{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}noscript{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}iframe{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}hr{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:calc(1.45rem - 1px);background:hsla(0,0%,0%,0.2);border:none;height:1px;}address{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}b{font-weight:bold;}strong{font-weight:bold;}dt{font-weight:bold;}th{font-weight:bold;}li{margin-bottom:calc(1.45rem / 2);}ol li{padding-left:0;}ul li{padding-left:0;}li > ol{margin-left:1.45rem;margin-bottom:calc(1.45rem / 2);margin-top:calc(1.45rem / 2);}li > ul{margin-left:1.45rem;margin-bottom:calc(1.45rem / 2);margin-top:calc(1.45rem / 2);}blockquote *:last-child{margin-bottom:0;}li *:last-child{margin-bottom:0;}p *:last-child{margin-bottom:0;}li > p{margin-bottom:calc(1.45rem / 2);}code{font-size:0.85rem;line-height:1.45rem;}kbd{font-size:0.85rem;line-height:1.45rem;}samp{font-size:0.85rem;line-height:1.45rem;}abbr{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;}acronym{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;}abbr[title]{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;text-decoration:none;}thead{text-align:left;}td,th{text-align:left;border-bottom:1px solid hsla(0,0%,0%,0.12);font-feature-settings:\\"tnum\\";-moz-font-feature-settings:\\"tnum\\";-ms-font-feature-settings:\\"tnum\\";-webkit-font-feature-settings:\\"tnum\\";padding-left:0.96667rem;padding-right:0.96667rem;padding-top:0.725rem;padding-bottom:calc(0.725rem - 1px);}th:first-child,td:first-child{padding-left:0;}th:last-child,td:last-child{padding-right:0;}",
              }
            }
            id="typography.js"
          />
          <link
            href="//fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i"
            rel="stylesheet"
            type="text/css"
          />
          <div
            className="Button "
            onClick={[Function]}
          >
            <h4
              className="Button-label"
            >
              CLICK ME
            </h4>
          </div>
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": "auto",
              "height": 20,
              "left": -19,
              "position": "absolute",
              "right": "auto",
              "top": -19,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 5,
                "top": undefined,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 5,
                "left": "auto",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": "auto",
              "height": 20,
              "left": "auto",
              "position": "absolute",
              "right": -19,
              "top": -19,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 5,
                "position": "absolute",
                "right": 0,
                "top": "auto",
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 5,
                "left": 0,
                "position": "absolute",
                "right": "auto",
                "top": 0,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": -19,
              "height": 20,
              "left": -19,
              "position": "absolute",
              "right": "auto",
              "top": "auto",
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": "auto",
                "left": 0,
                "position": "absolute",
                "right": 5,
                "top": 0,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": "auto",
                "position": "absolute",
                "right": 0,
                "top": 5,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": -19,
              "height": 20,
              "left": "auto",
              "position": "absolute",
              "right": -19,
              "top": undefined,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": "auto",
                "left": 5,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": "auto",
                "top": 5,
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Button with lots of text 1`] = `
<div
  data-radium={true}
  style={
    Object {
      "background": "rgb(255, 255, 255)",
      "bottom": 0,
      "display": "flex",
      "flexDirection": "column",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    data-radium={true}
    style={
      Object {
        "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 15,
        "paddingBottom": 15,
        "paddingTop": 2,
      }
    }
  >
    <h2
      data-radium={true}
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.6)",
          "fontSize": 20,
          "fontWeight": 200,
          "margin": 0,
          "padding": 0,
        }
      }
    >
      Kleros UI-Kit
    </h2>
  </div>
  <div
    data-radium={true}
    style={
      Object {
        "boxSizing": "border-box",
        "flex": 1,
        "margin": 50,
        "position": "relative",
      }
    }
  >
    <div
      data-radium={true}
      style={
        Object {
          "alignItems": "center",
          "bottom": 0,
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "center",
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <div
        data-radium={true}
        style={
          Object {
            "border": "solid 1px rgba(0, 0, 0, 0.0)",
            "boxSizing": "border-box",
            "height": "auto",
            "position": "relative",
            "width": "auto",
          }
        }
      >
        <div
          data-radium={true}
        >
          <style
            dangerouslySetInnerHTML={
              Object {
                "__html": "html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block}audio:not([controls]){display:none;height:0}progress{vertical-align:baseline}[hidden],template{display:none}a{background-color:transparent;-webkit-text-decoration-skip:objects}a:active,a:hover{outline-width:0}abbr[title]{border-bottom:none;text-decoration:underline;text-decoration:underline dotted}b,strong{font-weight:inherit;font-weight:bolder}dfn{font-style:italic}h1{font-size:2em;margin:.67em 0}mark{background-color:#ff0;color:#000}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}img{border-style:none}svg:not(:root){overflow:hidden}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}figure{margin:1em 40px}hr{box-sizing:content-box;height:0;overflow:visible}button,input,optgroup,select,textarea{font:inherit;margin:0}optgroup{font-weight:700}button,input{overflow:visible}button,select{text-transform:none}[type=reset],[type=submit],button,html [type=button]{-webkit-appearance:button}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-cancel-button,[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-input-placeholder{color:inherit;opacity:.54}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}html{font:100%/1.45em 'Roboto';box-sizing:border-box;overflow-y:scroll;}*{box-sizing:inherit;}*:before{box-sizing:inherit;}*:after{box-sizing:inherit;}body{color:hsla(0,0%,0%,0.8);font-family:'Roboto';font-weight:normal;word-wrap:break-word;font-kerning:normal;-moz-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";-ms-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";-webkit-font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";font-feature-settings:\\"kern\\", \\"liga\\", \\"clig\\", \\"calt\\";}img{max-width:100%;margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}h1{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:2rem;line-height:1.1;}h2{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1.51572rem;line-height:1.1;}h3{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1.31951rem;line-height:1.1;}h4{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:1rem;line-height:1.1;}h5{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:0.87055rem;line-height:1.1;}h6{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;color:inherit;font-family:'Roboto';font-weight:bold;text-rendering:optimizeLegibility;font-size:0.81225rem;line-height:1.1;}hgroup{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}ul{margin-left:1.45rem;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;list-style-position:outside;list-style-image:none;}ol{margin-left:1.45rem;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;list-style-position:outside;list-style-image:none;}dl{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}dd{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}p{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}figure{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}pre{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;font-size:0.85rem;line-height:1.45rem;}table{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;font-size:1rem;line-height:1.45rem;border-collapse:collapse;width:100%;}fieldset{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}blockquote{margin-left:1.45rem;margin-right:1.45rem;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}form{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}noscript{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}iframe{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}hr{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:calc(1.45rem - 1px);background:hsla(0,0%,0%,0.2);border:none;height:1px;}address{margin-left:0;margin-right:0;margin-top:0;padding-bottom:0;padding-left:0;padding-right:0;padding-top:0;margin-bottom:1.45rem;}b{font-weight:bold;}strong{font-weight:bold;}dt{font-weight:bold;}th{font-weight:bold;}li{margin-bottom:calc(1.45rem / 2);}ol li{padding-left:0;}ul li{padding-left:0;}li > ol{margin-left:1.45rem;margin-bottom:calc(1.45rem / 2);margin-top:calc(1.45rem / 2);}li > ul{margin-left:1.45rem;margin-bottom:calc(1.45rem / 2);margin-top:calc(1.45rem / 2);}blockquote *:last-child{margin-bottom:0;}li *:last-child{margin-bottom:0;}p *:last-child{margin-bottom:0;}li > p{margin-bottom:calc(1.45rem / 2);}code{font-size:0.85rem;line-height:1.45rem;}kbd{font-size:0.85rem;line-height:1.45rem;}samp{font-size:0.85rem;line-height:1.45rem;}abbr{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;}acronym{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;}abbr[title]{border-bottom:1px dotted hsla(0,0%,0%,0.5);cursor:help;text-decoration:none;}thead{text-align:left;}td,th{text-align:left;border-bottom:1px solid hsla(0,0%,0%,0.12);font-feature-settings:\\"tnum\\";-moz-font-feature-settings:\\"tnum\\";-ms-font-feature-settings:\\"tnum\\";-webkit-font-feature-settings:\\"tnum\\";padding-left:0.96667rem;padding-right:0.96667rem;padding-top:0.725rem;padding-bottom:calc(0.725rem - 1px);}th:first-child,td:first-child{padding-left:0;}th:last-child,td:last-child{padding-right:0;}",
              }
            }
            id="typography.js"
          />
          <link
            href="//fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i"
            rel="stylesheet"
            type="text/css"
          />
          <div
            className="Button "
            onClick={[Function]}
          >
            <h4
              className="Button-label"
            >
              CLICK ME, CLICK ME, CLICK ME
            </h4>
          </div>
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": "auto",
              "height": 20,
              "left": -19,
              "position": "absolute",
              "right": "auto",
              "top": -19,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 5,
                "top": undefined,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 5,
                "left": "auto",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": "auto",
              "height": 20,
              "left": "auto",
              "position": "absolute",
              "right": -19,
              "top": -19,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 5,
                "position": "absolute",
                "right": 0,
                "top": "auto",
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 5,
                "left": 0,
                "position": "absolute",
                "right": "auto",
                "top": 0,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": -19,
              "height": 20,
              "left": -19,
              "position": "absolute",
              "right": "auto",
              "top": "auto",
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": "auto",
                "left": 0,
                "position": "absolute",
                "right": 5,
                "top": 0,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": "auto",
                "position": "absolute",
                "right": 0,
                "top": 5,
              }
            }
          />
        </div>
        <div
          data-radium={true}
          style={
            Object {
              "bottom": -19,
              "height": 20,
              "left": "auto",
              "position": "absolute",
              "right": -19,
              "top": undefined,
              "width": 20,
            }
          }
        >
          <div
            data-radium={true}
            style={
              Object {
                "borderBottom": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": "auto",
                "left": 5,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          />
          <div
            data-radium={true}
            style={
              Object {
                "borderRight": "solid 1px rgba(0, 0, 0, 0.1)",
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": "auto",
                "top": 5,
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
